test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+const fs = require( 'fs/promises' );
+import os from 'os';
+import { v4 as uuid } from 'uuid';
```

Comment:
Sorry, this was my complete oversight in copy-paste. 

One suggestion, given that `require` import is not appropriate, a lint rule may be useful to catch this.

Reply:
Fixed in d6a4f4499555c768f60a9ca86ae70cc0746e1332.

Reply:
I mean use `require` not `import` 😅 . We have to use `require` for now because of some ESM and NodeJS inconsistency 😞 .

Reply:
> I mean use `require` not `import` 😅 . We have to use `require` for now because of some ESM and NodeJS inconsistency 😞 .

Interesting. The spec seems to run fine in both local and CI environments, but I'll change it over to `require` in the next revision.

Reply:
Changed to `require` in 7897f28b7eaa3146eb2a023d536bc98c437eb884.